mailpoet/assets/js/src/newsletters/send/congratulate/success_pitch_mss.tsx

```diff
@@ -3,10 +3,19 @@ import { MailPoet } from 'mailpoet';
 
 import { Heading } from 'common/typography/heading/heading';
 import { WelcomeWizardStepLayoutBody } from '../../../wizard/layout/step_layout_body.jsx';
-import {
-  Controls,
-  FreeBenefitsList,
-} from '../../../wizard/steps/pitch_mss_step';
+import { List } from '../../../common/typography/list/list';
+import { Button } from '../../../common';
+
+function FreeBenefitsList(): JSX.Element {
```

Comment:
Suggestion: I'd leave the type inference to TS in cases it can infer the type, and this is one these cases.

Reply:
Good to know that this is an option. Due to time constraints, and since I assume having the type is not an issue, I'm not planning to implement this change in this PR (please let me know if you think that I should). But I will keep this in mind going forward.